ansible/roles/hadoop/tasks/main.yml

```diff
@@ -34,6 +34,11 @@
   with_items:
     - workers
   when: hadoop_major_version == '3'
+
+- name: "Copy javax.activation-api (Hadoop 3)"
+  synchronize: src={{ user_home }}/mvn_dep/ dest={{ hadoop_home }}/share/hadoop/common/lib/
```

Comment:
Will this copy the file on the current machine?  Would be nice to add a comment saying this is currently needed to run hadoop with Java 11.

Reply:
Will do, Keith. I am still testing this overall, and found some interesting issues with Java 11 SDK JAVA_HOME etc.; allow me some time before I mark this as a final PR. And to your question, synchronize uses rsync internally to copy any file(s) from src folder on the proxy to the destination folder on the worker host(s). For just 1 file, rsync may not be faster, but let's say in the future we have more such dependencies, we found that the synchronize module is faster than the copy module.

Reply:
I was wondering if it would do the sync on the proxy machine (to itself), assuming Hadoop is on the proxy machine.  I took a look at the docs for the module and could not tell.

Reply:
Ah. I can see your potential confusion. No worries, synchronize will by default copy from the Ansible controller (in Muchos, that is the "proxy") to the host.